SourceGitlab/README.md

```diff
@@ -7,19 +7,19 @@
 
 ## Gitlab setup
 
- - Login with an Owner (or Adminstrator) of a Project, go to "Projet settings", "Web hooks",
-   add a Push hook with Mantis url and Mantis Source Integration plugin "API Key":
-   http://mantis.server.intra/plugin.php?page=Source/checkin&api_key=abcdeb8129a4451a35f47881
- - Create a user with (at least) read access to the repository. For a public repository, any user would work.
-   Login with this user, go to "Profile settings", "Account", and copy the "Private token".
+ - Login with an Owner (or Adminstrator) of a Project, go to "Settings" -> "Integrations" and
```

Comment:
- _Administrator_
- Suggest to use MarkDown italics instead of using quotes, e.g. `_Settings -> Integrations_ `

Reply:
done